docs/guides/development/data-synchronization.md

```diff
@@ -0,0 +1,169 @@
+
+### Customer synchronization options
+
+<!-- https://success.voucherify.io/article/430-crm-playbook
+Adding customers
+
+These are the ways you can add customer profiles to Voucherify:
+
+API
+Webhooks
+The import with a CSV file. 
+You can integrate your CRM platform with Voucherify.
+
+ -->
+
+#### Upserting customers
+
+If the customer does not exist in Voucherify, they are created automatically in a request that includes the customer object. If the customer exists, their data will be upserted. In this case, the response also includes the `updated_at` object. <!-- these are objects, right? -->
+
+The following actions upsert customer data:
+- [Order creation](ref:create-order)
+- [Stackable discount redemption](ref:redeem-stacked-discounts)
+- [Stackable discount redemption (client side)](ref:redeem-stacked-discounts-client-side)
+- [Track custom event](ref:track-custom-event)
+- [Voucher publication](ref:create-publication)
+
+#### Importing customers
+
+You can import your customer database to Voucherify with a CSV file. If you use a CRM system, use its unique ID as a `source_id` field.
+
+Read our [customer import guide](https://support.voucherify.io/article/67-how-to-import-my-customers#csv-import "How to import my customers?") to learn more.
+
+<!-- Should I write a full step-by-step manual for this? -->
+
+#### Creating and updating customers via the API
+
+If you want to keep your customers database up to date with Voucherify, create a customer in Voucherify every time a new user is added to your database. Use the [Create customer](ref:create-customer) API endpoint to create customer data.
+
+To update customer data, use [Update customer](ref:update-customer) API endpoint. The [Create customer](ref:create-customer) API endpoint can be also used to update customer data.
+
+<!-- Simply repeat the endpoints? -->
+
+#### Synchronizing with connectors
+
+Customer data can be upserted with integrated platforms that support outbound traffic:
+- [Bloomreach Engagement](https://support.voucherify.io/article/613-bloomreach-engagement-integration "Voucherify-Bloomreach Engagement integration article")
+- [Braze](https://support.voucherify.io/article/588-braze-integration "Voucherify-Braze integration article")
+- [mParticle](https://support.voucherify.io/article/590-mparticle "Voucherify-mParticle integration article")
+- [Salesforce](https://support.voucherify.io/article/140-salesforce "Voucherify-Salesforce integration article")
+- [Segment](https://support.voucherify.io/article/272-segment "Voucherify-Segment integration article")
+- [Zapier](https://support.voucherify.io/article/269-zapier "Voucherify-Zapier integration article")
+
+### Customer API endpoints
+
+Visit our API reference to see the list of [customer endpoints](ref:customer-object) that can be used to synchronize customer data.
+
+> 📘 Upserting customer data through API
+>
+> All endpoints that have a customer object can be used to upsert customer data.
```

Comment:
No, this one is not 100% true. I think we can remove this section, since it's copying lines 64-69

Reply:
Ok, removed